loadtesting/k6/data/accounts.json

```diff
@@ -0,0 +1,12 @@
+{
+    "0": {
+        "sender_address": "0x64f387a5b5fAe44C1fC00a5FB5328E09a94166BE",
```

Comment:
I think this file is committed accidentally

Reply:
fixed

Reply:
k6 data folder is also added to gitignor